std/typecons.d

```diff
@@ -4892,7 +4892,7 @@ to deallocate the corresponding resource.
             GC.removeRange(&_refCounted._store._payload);
         }
         import core.stdc.stdlib : free;
-        free(_refCounted._store);
+        () @trusted { free(_refCounted._store); }();
```

Comment:
I am not sure about this one. Is it safe to free memory from a pointer that we created and keep the reference too?